readme.md

```diff
@@ -1,48 +1,116 @@
-- Update the bin/postInstall script. It should refer to your library.
-- Update the `<title>` tag in `index.html.js`.
-- Remove 'BEFORE YOU PUBLISH' and 'PUBLISHING' from this document.
-
-# PUBLISHING
-- Make sure you are added to the kaliber organization on NPM
-- run `yarn publish`
-- Enter a correct version, we adhere to semantic versioning (semver)
-- run `git push`
-- run `git push --tags`
-- Send everybody an email to introduce them to your library!
-
-# Library title
-Short description.
+# @kaliber/math
+A set of useful math functions! 
 
 ## Motivation
-Optionally add a bit of text describing why this library exists.
+Some math problems are just way easier to reason about if you have access to some higher level abstractions. This library provides some of these abstractions in a clear, consistent API.
 
 ## Installation
 
 ```
-yarn add @kaliber/library
+yarn add @kaliber/math
 ```
 
-## Usage
-Short example. If your library has multiple ways to use it, show the most used one and refer to `/example` for further examples.
+## Reference
 
-```jsx
-import { hello } from 'library'
+1. [`lerp`](#lerp)
+2. [`unlerp`](#unlerp)
+3. [`clamp`](#clamp)
 
-function Component() {
-  return <div>{hello()}</div>
-}
+### `lerp`
+`lerp({ start: number, end: number, amount: number, clamp: boolean = false })`
+
+Linearly interpolates between points.
+
+| Argument | Type |  |
+| --- | --- | --- |
+| `start` | `Number` | Start of the input range. |
+| `end` | `Number` | End of the input range. |
+| `input` | `Number` | Input (usually between `0` and `1`), indicating where to interpolate. |
+| `clamp` | `Boolean` (default is `false`) | Whether to clamp the output value between the `start` and `end` values. |
+
+```js
+const rotZ = lerp({ start: -45, end: 45, input: 0.5 })
+```
+
+### `unlerp`
+`unlerp({ start: number, end: number, input: number, clamp: boolean = false })`
+
+Normalizes a number to a value between 0 and 1, given an input range.
+
+| Argument | Type |  |
+| --- | --- | --- |
+| `start` | `Number` | Start of the input range. |
+| `end` | `Number` | End of the input range. |
+| `input` | `Number` | The value to normalize. |
+| `clamp` | `Boolean` (default is `false`) | Whether to clamp the output value between `0` and `1`. |
+
+```js
+const normalizedMouseX = unlerp({ start: 0, end: window.innerWidth, input: mouseX })
+```
+
+### `clamp`
+`clamp({ min: number, max: number, input: number })`
+
+Constrain a number between two boundary values. The boundary arguments are called `min` and `max` to make them easy to memorize, but swapping them has no consequences.
+
+| Argument | Type |  |
+| --- | --- | --- |
+| `min` | `Number` | First boundary value |
+| `max` | `Number` | Second boundary value |
+| `input` | `Number` | Number to constrain |
+
+```js
+const clamped = clamp({ min: 0, max: 1, input: normalizedMouseX })
+```
+
+```js
+const clamped = clamp({ min: 1, max: 0, input: Math.random() })
+```
```

Comment:
Ik zou er geen voorbeeld bij zetten dat niet intuitief is: `min` op `1` en `max` op `0` klinkt raar. Ik neem aan dat je er een use case voor hebt om `min` en `max` kwa volgorde niet uit te laten maken. 
```suggestion
```